fireplace/cards/wog/druid.py

```diff
@@ -4,4 +4,10 @@
 ##
 # Minions
 
+class OG_313:
+	"Addled Grizzly"
+	events = Summon(CONTROLLER, MINION - SELF).after(
```

Comment:
I just copied this from Knife Juggler - figured there would be a reason for it. All tests pass after removing it from Knife Juggler too though.